Mathlib/Algebra/Category/Ring/Basic.lean

```diff
@@ -126,6 +126,10 @@ instance : ConcreteCategory.{u} SemiRingCat where
       map := fun f => f.hom }
   forget_faithful := ⟨fun h => by ext x; simpa using congrFun h x⟩
 
+/-- This unification hint helps with problems of the form `(forget ?C).obj R =?= carrier C`. -/
+unif_hint forget_obj_eq_coe (R : SemiRingCat) where ⊢
+  (forget SemiRingCat).obj R ≟ SemiRingCat.carrier R
```

Comment:
I think the formatting is supposed to be
```suggestion
unif_hint forget_obj_eq_coe (R : SemiRingCat) where
  ⊢ (forget SemiRingCat).obj R ≟ SemiRingCat.carrier R
```

Is the docstring correct? I'm confused about `?C` vs `C`. Should the second `C` be an `R`?

I'm also wondering how this compares to say
```lean
unif_hint forget_obj_eq_coe where
  ⊢ (forget SemiRingCat.{u}).obj ≟ SemiRingCat.carrier
```
or to
```lean
unif_hint forget_obj_eq_coe (R R' : SemiRingCat) where
  R ≟ R'
  ⊢ (forget SemiRingCat).obj R ≟ SemiRingCat.carrier R'
```